modules/python/test/test_norm.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+from itertools import product
+from functools import reduce
+
+import cv2 as cv
+import numpy as np
```

Comment:
Please import `numpy` before `cv2` (there are some reports related to importing order).